File: 3246-CheckIfBitwiseOrHasTrailingZeros/3246-CheckIfBitwiseOrHasTrailingZeros.py
# Last updated: 31/07/2025, 21:05:55
class Solution:
    def hasTrailingZeros(self, nums: List[int]) -> bool:
        c=0
        for i in nums:
            if i%2==0:
                c+=1   
        if c>1:
            return True
        else:
            return False

File: 3246-CheckIfBitwiseOrHasTrailingZeros/3246-CheckIfBitwiseOrHasTrailingZeros.java
// Last updated: 31/07/2025, 21:05:51
class Solution {
    public boolean hasTrailingZeros(int[] nums) {
        int c=0;
        for(int i=0;i<nums.length;i++)
        {
            if (nums[i]%2==0)
            {
                c+=1;
                if (c>=2)
                {
                    return true;
                }
        }
        
    }
    return false;
}
}